src/main/java/io/github/a1qs/vaultadditions/config/vault/AbstractStatueLootConfig.java

```diff
@@ -28,6 +29,14 @@ protected void reset() {
 
     }
 
+    public WeightedList<ProductEntry> getDrops() {
+        return DROPS;
+    }
```

Comment:
Can you make this immutable, either sift with a wrapped list or hard if there's an implementation for that already